lib/jira.py

```diff
@@ -33,6 +35,8 @@ def search_issues():
     parser.add_argument('-u', '--url', help='URL to Jira', required=True)
     parser.add_argument('-q', '--query', help='Jira qrl query', required=True)
     parser.add_argument('-c', '--credentials-file', help='Path to credentials file', required=True)
+    parser.add_argument('-r', '--release', help='Path to current release file', required=True)
+    parser.add_argument('-p', '--previousRelease', help='Path to previous release file', required=True)
```

Comment:
I would go with something like that.

```suggestion
    parser.add_argument('-r', '--release', help='Path to current release file. Not used, supported to align the interface.', required=False)
    parser.add_argument('-p', '--previousRelease', help='Path to previous release file. Not used, supported to align the interface.', required=False)
```